pages/rs_quality/fair.md

```diff
@@ -1,5 +1,178 @@
+
+### Persistent identifiers
+
+Unique persistent identifiers, such as Digital Object Identifiers (DOIs) provided by Zenodo, FigShare and similar digital archiving services, and commits/tags/releases used by GitHub and similar code sharing platforms, 
+help with findability and accessibility of your software, and can help you get credit for your work by providing citable references.
+
+### Tools for assessing FAIRness of software
+
+Here are some tools that can check your software and provide an assessment or measurement of its FAIRness:
+
+- [FAIRsoft evaluator][fair-rs-evaluator]
+- [FAIR software test][fair-rs-test]
+
+The table below provides a summary of how different tools and practices help with the FAIR software principles.
+
+| Tools and practices                                                                                  | Findable | Accessible | Interoperable | Reusable |
+| ---------------------------------------------------------------------------------------------------- | -------- | ---------- | ------------- | -------- |
+| Virtual development environments                                                                     |          | x          |               | x        |
+| Integrated development environments/IDEs                                                             |          |            |               | x        |
+| Command line terminals - automated and reproducible pipelines                                        |          |            | x             | x        |
+| Standard formats - e.g. for data exchange (CSV, YAML)                                                |          | x          | x             | x        |
+| Communication protocols - Command Line Interface (CLI) or Application Programming Interface (API)    |          | x          | x             | x        |
+| Version control tools                                                                                | x        |            |               |          |
+| Testing - code correctness and reproducibility                                                       |          | x          |               | x        |
+| Coding conventions                                                                                   |          | x          | x             | x        |
+| Software-level documentation (comments and docstrings, explaining functionality)                     |          | x          | x             | x        |
+| Project-level documentation (READMEs, explaining functionality/installation/running)                 |          | x          | x             | x        |
+| License - code sharing and reuse                                                                     |          | x          |               | x        |
+| Citation - code reuse and credit                                                                     | x        |            |               | x        |
```

Comment:
This is reusable, I believe, rather than findable. It's the provenance of the software component. We did not mention citation.cff, which is another task in itself.

Reply:
Sorry, where is CFF mentioned in the page? Maybe I did not see it?

Reply:
Oh, my bad, you mention citation in general. Maybe it should be a subsection too? (for consistency with the rest)

Reply:
Fixed via 88db083e4868b269066ee48e625ad94e198f646c.